docs/modules/internal_modules.md

```diff
@@ -0,0 +1,85 @@
+# List of Modules
+
+## What are internal modules?
+
+Internal modules just like regular modules, except that they run all the time. They do not have to be explicitly enabled. They can, however, be explicitly disabled if needed.
+
+Turning them off is simple, a root-level config option is present which can be set to False to disable them:
+
+```
+# Infer certain events from others, e.g. IPs from IP ranges, DNS_NAMEs from URLs, etc.
+speculate: True
+# Passively search event data for URLs, hostnames, emails, etc.
+excavate: True
+# Summarize activity at the end of a scan
+aggregate: True
+# DNS resolution
+dnsresolve: True
+# Cloud provider tagging
+cloudcheck: True
+```
```

Comment:
Todo: fix these before merge

Reply:
Whats the issue here?

Reply:
Some of the module names aren't right but that's my fault; I'll handle changing them.